pkg/monitoring/metrics/virt-controller/metrics.go

```diff
@@ -143,3 +147,13 @@ func getTransitionTimeSeconds(oldTime *metav1.Time, newTime *metav1.Time) (float
 
 	return diffSeconds, nil
 }
+
+func SetupPVCInformer() {
+	// Adding the PVC index setup by namespace and name
+	persistentVolumeClaimInformer.AddIndexers(cache.Indexers{
```

Comment:
Why would you need this? Informers already cache the resource by namespace/name. Should be able to do something like:
```golang
persistentVolumeClaimInformer.GetStore().GetByKey("ns/name")
```

Reply:
thanks a lot finally able to get it working! now adding more labels